src/NexusMods.App.UI/Pages/LoadoutGrid/LoadoutGridViewModel.cs

```diff
@@ -83,13 +74,8 @@ public LoadoutGridViewModel() : base(null!)
         IWindowManager windowManager,
         ISettingsManager settingsManager) : base(windowManager)
     {
-        _logger = logger;
-        _fileSystem = fileSystem;
         _conn = conn;
-        _archiveInstaller = archiveInstaller;
-        _fileOriginRegistry = fileOriginRegistry;
-        _provider = provider;
-        
```

Comment:
You can remove fileSystem, archiveInstaller and fileOriginRegistry parameters from the constructor as well